hs_core/views/resource_folder_hierarchy.py

```diff
@@ -91,11 +86,21 @@ def data_store_structure(request):
                 folder_aggregation_type = aggregation_object.get_aggregation_class_name()
                 folder_aggregation_name = aggregation_object.get_aggregation_display_name()
                 folder_aggregation_id = aggregation_object.id
-                if not aggregation_object.is_fileset:
+                if not aggregation_object.is_fileset and not aggregation_object.is_model_program and \
```

Comment:
fileset is handled similar to model program/instance a lot.  I believe I mentioned this above, but I think instead of having to check for all the types each time, we should just check a property on the aggregation.

Reply:
Removed check on different file types here